core/src/main/java/de/flo56958/MineTinker/Utilities/ChatWriter.java

```diff
@@ -25,6 +25,12 @@
     static {
     	CHAT_PREFIX = Main.getPlugin().getConfig().getString("chat-prefix");
 
+        map.put(1000000, "%BOLD%%UNDERLINE%M");
```

Comment:
You need to reset it after the Character as the formatting will persist with smaller (not bold) characters.